Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftDefenderEventCollector/MicrosoftDefenderEventCollector.yml

```diff
@@ -6,6 +6,18 @@ commonfields:
   id: Microsoft Defender for Cloud Apps Event Collector
   version: -1
 configuration:
+- additionalinfo: |-
+    More information can be found on the integration page - https://xsoar.pan.dev/docs/reference/integrations/microsoft-defender-for-cloud-apps-event-collector
+  display: Endpoint Type
+  name: endpoint_type
+  required: false
+  type: 15
+  section: Connect
+  defaultvalue: Worldwide
+  options:
+  - Worldwide
+  - US GCC
+  - US GCC-High
```

Comment:
Could we allow a Custom option like in defender-atp?

Reply:
no need for it, for this product.